src/Event/Http/HttpResponse.php

```diff
@@ -20,48 +16,40 @@ final class HttpResponse
     /** @var string */
     private $body;
 
-    public function __construct(string $body, array $headers, int $statusCode = 200)
+    public function __construct(string $body, array $headers = [], int $statusCode = 200)
```

Comment:
This class will now be used by end-users. This helps making it more developer-friendly.